technical/validations.md

```diff
@@ -7,20 +7,49 @@ these validations will be duplicated in the front-end code for an optimal user e
 ## Entity-level validations
 
 ### Playground
-- The name of the playground is between 3 en 40 characters
-- Two playgrounds cannot be located within .. metres of each other
-- The maximum number of playgrounds that can exist within the system is ..
+- the name of the playground is between 3 en 40 characters
+- two playgrounds can not have the same name
+- no whitespaces in the beginning and trailing of the playground name
+- two playgrounds cannot be located within 100 metres of each other
+- the maximum number of playgrounds that can exist within the system is 1000
+- at least 4 decimal precisions required for longitude and latitude of the playground location, in order to avoid duplicate entries
+- a user can join the playground where he is already a volunteer in, then system just returns success and ignore the request
+- the id, name, lat, lng and status are required properties during creation of a new playground
+- for onboarding smokefree playground, the smokeFreeDate must be past
```

Comment:
onboarding smokefree playground -> onboarding playground with status 'finished' (that claims to be smokefree)